app/layout/community/published-projects/duplicate-button/component.tsx

```diff
@@ -71,12 +75,17 @@ export const DuplicateButton: React.FC<DuplicateButtonProps> = ({
     <Button
       className="px-6 group"
       size="s"
-      theme="transparent-white"
+      theme={theme === 'light' ? 'transparent-white' : 'transparent-black'}
```

Comment:
You can use classnames package for anything you want. I probably hate ternaries...
```suggestion
      theme={classnames({
        'transparent-white': theme === 'light',
        'transparent-black': theme !== 'light' 
      })}
```